### ISSUE TITLE
Sheet can't paste image 

### ISSUE THREAD

**Issue by Keeyuu** [cryptpad/cryptpad#1429]
### Description

I tried other formats like documents and markdown, and they worked just fine. It's only with spreadsheets that I'm having an issue. Here's what I'm doing: I copy an image to the clipboard using ctrl+c, then in the spreadsheet, I try to paste it using ctrl+v. But it doesn't just paste the image directly onto the sheet like it does in a document. And no window pops up to let me upload the image either.

### Steps to reproduce

1. I copy an image to the clipboard using ctrl+c
2. then in the spreadsheet
3. I try to paste it using ctrl+v. But it doesn't just paste the image directly onto the sheet like it does in a document. And no window pops up to let me upload the image either.

### Relevant logs

```shell
no
```

### Expected behavior

no

### Which operating system are you using?

Linux/BSD/UNIX

### What browsers are you seeing the problem on?

Microsoft Edge

### Extensions

no

### Version

5.7.0

**Comment by Keeyuu** [comment]
I would like to inquire about which part of the code need to be modified in order to implement this additional feature. I would like to manually implement it first.

**Comment by Keeyuu** [comment]
> Hello @Keeyuu, thanks for your interest in contributing to CryptPad!
> 
> This is a difficult one. CryptPad uses a modified version of OnlyOffice. The code that handles the clipboard is here: https://github.com/cryptpad/sdkjs/blob/cp7.3.3.60/cell/model/clipboard.js#L2804 . As you see the pasting of images is not implemented and disabled by us.
> 
> Pasted images have to be uploaded into CryptPad first and then inserted into OnlyOffice.

Thanks for your patient guidance

**Comment by ghost** [comment]
Issue is not specific for spreadsheet, but also applies to Document, Presentation and Diagram.
In Diagram the behavior is slightly different, this pastes a link when the paste buffer contains a picture copied from the internet. 

An relatively easy workaround might be to add an option to add a 'Insert from URL" in the _Insert / Image/_ menu, in addition to   _Image from file_

**Comment by filipesmedeiros** [comment]
I agree that adding an image from URL would be a good workaround, because the main use case is when you want to paste an image from another tab of the browser, in which case it will have a URL that you can bring over to CryptPad. :)
